### PR TITLE
Fix: mobile menu button centering + a link in tx breadcrumbs

### PR DESCRIPTION
**Pull request by katspaugh**
Two small fixes:

* Menu button should be flex for centering
* Added a link to all txns from the single tx breadcrumbs

### CODE REVIEW

File: src/pages/safe/transactions/tx.tsx [hunk 1]
```diff
@@ -50,14 +51,15 @@ const SingleTransaction: NextPage = () => {
   )
 
   const isCurrentSafeTx = sameAddress(txDetails?.safeAddress, safeAddress)
+  const breadcrumbsLink = `${AppRoutes.safe.transactions.index}?safe=${safeAddress}`
```
Comment (iamacook) [inline src/pages/safe/transactions/tx.tsx:54]: Perhaps we could automate by remove everything after the last `/`?

**Review by iamacook — COMMENTED** [review]
Looks good to me but it needs to be added to the whole transactions section, as well as the settings area.

File: src/pages/safe/transactions/tx.tsx (hunk 1 above)
Comment (katspaugh) [reply src/pages/safe/transactions/tx.tsx:54]: I’d go with this custom link for now, but good idea for another PR.

**Comment by katspaugh** [comment]
@iamacook noch einen Wunsch? :)

**Comment by iamacook** [comment]
> @iamacook noch einen Wunsch? :)

We should add the `firstLink` for all transaction breadcrumbs and the settings as well.

**Comment by katspaugh** [comment]
Like I said, not in this PR.

**Review by iamacook — APPROVED** [review]

**Pull request merged**